numpy/core/tests/test_multiarray.py

```diff
@@ -7486,7 +7486,7 @@ def test_out_of_order_fields(self):
             memoryview(arr)
 
     def test_max_dims(self):
-        a = np.empty((1,) * 32)
```

Comment:
Was there a problem with `empty`? I agree `ones` is nicer.

Reply:
Its a harmless "use of uninitialized memory" in valgrind because `check_roundtrip` actually does something with the values (or at least it looks like it).